plugin/pkg/scheduler/scheduler_test.go

```diff
@@ -273,8 +253,7 @@ func TestSchedulerForgetAssumedPodAfterDelete(t *testing.T) {
 
 	// Second scheduling pass will fail to schedule if the store hasn't expired
 	// the deleted pod. This would normally happen with a timeout.
-	//expirationPolicy.NeverExpire = util.NewStringSet()
-	fakeClock.Step(ttl + 1)
+	time.Sleep(2 * time.Second)
```

Comment:
I don't like Sleeps in tests, cause there were causing a bunch of flakes.
Can we change it to use some WaitGroup{} or sth like that?